src/wp-includes/class-wpdb.php

```diff
@@ -694,13 +691,17 @@ class wpdb {
 	private $allow_unsafe_unquoted_parameters = true;
 
 	/**
-	 * Whether to use mysqli over mysql. Default false.
+	 * Whether to use the mysqli extension over mysql. This is no longer used as the mysql
+	 * extension is no longer supported.
+	 *
+	 * Default true.
 	 *
 	 * @since 3.9.0
+	 * @since x.y.z The default value was changed to true and the property is no longer used.
 	 *
 	 * @var bool
 	 */
-	private $use_mysqli = false;
+	private $use_mysqli = true;
```

Comment:
Is it even worth keeping this? While I get that we're generally careful with removing things, given that it's a private property, I don't think there is _any_ risk here? Not even a custom database class that extends `wpdb` would be able to use it, so since we're not using it anymore, I think the property could be removed.

Reply:
Good spot, and agreed. Thanks.

Reply:
Oops: https://core.trac.wordpress.org/ticket/59846